src/AWS.Deploy.Orchestration/TemplateEngine.cs

```diff
@@ -56,15 +56,17 @@ public async Task GenerateCDKProjectFromTemplate(Recommendation recommendation,
                 throw new Exception($"Failed to find a Template for [{recommendation.Recipe.CdkProjectTemplateId}]");
 
             var templateParameters = new Dictionary<string, string> {
-                { "AWSAccountID" , session.AWSAccountId },
-                { "AWSRegion" , session.AWSRegion },
-
                 // CDK Template projects can parameterize the version number of the AWS.Deploy.Recipes.CDK.Common package. This avoid
                 // projects having to be modified every time the package version is bumped.
                 { "AWSDeployRecipesCDKCommonVersion", FileVersionInfo.GetVersionInfo(typeof(Constants.CloudFormationIdentifier).Assembly.Location).ProductVersion
                                                       ?? throw new InvalidAWSDeployRecipesCDKCommonVersionException("The version number of the AWS.Deploy.Recipes.CDK.Common package is invalid.") }
             };
 
+            if (!string.IsNullOrEmpty(session.AWSAccountId))
+                templateParameters["AWSAccountID"] = session.AWSAccountId;
+            if (!string.IsNullOrEmpty(session.AWSRegion))
+                templateParameters["AWSRegion"] = session.AWSRegion;
```

Comment:
do we use `AWSAccountID` and `AWSRegion` template param? I thought we removed them and they are part of configs now.

Reply:
That is correct. It was handled as part of this PR - https://github.com/aws/aws-dotnet-deploy/pull/250